analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Operations.cs

```diff
@@ -472,11 +472,12 @@ public void Literal_Default_BoolLiterals(string literal)
         [DataRow("1.0f")]
         public void Literal_Default_OtherLiterals(string literal)
         {
-            var code = @$"
-var value = {literal};
-Tag(""Value"", value);";
+            var code = $$"""
```

Comment:
Can we just delete this test and move the values into the `RoslynSymbolicExecutionTest.Constant.cs` file among the other cases you've created?